project/SparkBuild.scala

```diff
@@ -748,6 +739,13 @@ object Java8TestSettings {
 object TestSettings {
   import BuildCommons._
 
+  private val scalaBinaryVersion =
+    if (System.getProperty("scala-2.10") == "true") {
```

Comment:
I don't really know. This is the same way the property is checked in `Java8TestSettings`, I just followed the pattern.